docs/UserGuide.md

```diff
@@ -93,11 +93,16 @@ The `Code Panel` allows users to see the code attributed to a specific author. C
 * Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the `...` icon.
 
 ### Commits Panel
+![Commits panel](/docs/images/commits-panel.png)
 
-The `Commits Panel` allows users to see the commits attributed to a specific author. Hold `Command`&#8984; (MacOS) or `Ctrl` (Others) and click on the ramp chart in the `Chart Panel` to select the time range where you want to display the `Commit Panel` for on the right.
-* The `Commits Panel` shows the commits that contain author's contributions, sorted by the date it was commited.
+The `Commits Panel` allows users to see the commits attributed to a specific author. Hold `Command`&#8984; (MacOS) or `Ctrl` (Others) and click on the ramp chart in the `Chart Panel` to select the time range where you want to display the `Commit Panel` for on the right. <br>
```

Comment:
Other OSes would be better, and perhaps bold them to attract attention.
Click on the ramp chart from one point to another to select the time range.